stepper-2.0.md

```diff
@@ -142,19 +166,24 @@ When a limit pin (digital) is set to its limit state, movement in that direction
 ```
 
 **Stepper set acceleration**
-Sets the acceleration/deceleration in steps/sec^2.
+
+Sets the acceleration/deceleration in steps/sec^2. Value is specified as step/sec^2 * 1000 since Firmata doesn't support sending and receiving floats.
+
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
 2  set acceleration command                (0x08)
 3  device number                           (0-9) (Supports up to 10 motors)
-4  accel LSB                               (acceleration in steps/sec^2)
-5  accel MSB
-6  END_SYSEX                               (0xF7)
+4  accel, bits 0-6                         (acceleration in steps/sec^2 * 1000)
```

Comment:
Lets go with 4 bytes here - same as speed. This shouldn't be called too frequently anyway (due to the square root calculate AccelStepper does when an acceleration value is set), so the 4th byte isn't a lot of overhead.

**accel**

|27   |26-23   |22-0       |
|-----|--------|-----------|
|sign |exponent|significand|
|1 bit|4 bits  |23 bits    |